madgiraffestudios/src/components/About/About.scss

```diff
@@ -2,6 +2,11 @@
     background-color: grayscale($color: #f8f9fa);
 }
 
+#about{
+
+    padding-bottom: .02px;
+}	
```

Comment:
can you please check this, I think it will cause a bug later. Or leverage pb-2 from bootstrap

Reply:
addressed